pennylane/templates/subroutines/amplitude_amplification.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This submodule contains the template for Amplitude Amplification.
+"""
+
+# pylint: disable-msg=too-many-arguments
+import numpy as np
+from pennylane.operation import Operation
+import pennylane as qml
+
+
+def get_fixed_point_angles(iters):
+    """
+    Returns the angles needed for the fixed-point amplitude amplification algorithm.
+    This is extracted from the equation (11) of the paper `fixed-point quantum search <https://arxiv.org/pdf/1409.3305.pdf>`__.
+    """
+    gamma = 0.95
```

Comment:
Is this number from the paper too?

Reply:
No, but I think it is overcomplicating to give this parameter to the user. It is a lower bound than the probability of success you want to achieve. Although it does not have to be met, it is an estimate of the lower bound. I was talking about it with Pablo and it is not unreasonable to leave it like this and if the user really asks for it at some point, add it to the template (this would not change the interface)